src/Command/Environment/EnvironmentDeployCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+namespace Platformsh\Cli\Command\Environment;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Cli\Console\AdaptiveTableCell;
+use Platformsh\Cli\Service\ActivityMonitor;
+use Platformsh\Client\Model\Activity;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class EnvironmentDeployCommand extends CommandBase
+{
+    private $tableHeader = [
+        'id' => 'ID',
+        'created' => 'Created',
+        'description' => 'Description',
+        'type' => 'Type',
+        'result' => 'Result',
+    ];
+
+    protected function configure()
+    {
+        $this
+            ->setName('environment:deploy')
+            ->setAliases(['deploy'])
+            ->setDescription('Deploy an environment');
+        $this->addProjectOption()
+            ->addEnvironmentOption();
+        $this->addWaitOptions();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->chooseEnvFilter = $this->filterEnvsByStatus(['active', 'paused']);
+        $this->validateInput($input);
+
+        $environment = $this->getSelectedEnvironment();
+
+        if (!$environment->operationAvailable('deploy', true)) {
+            $this->stdErr->writeln(
+                "Operation not available: The environment " . $this->api()->getEnvironmentLabel($environment, 'error') . " can't be deployed."
+            );
+
+            if (!$environment->isActive()) {
+                $this->stdErr->writeln('The environment is not active.');
+            }
+
+            return 1;
+        }
+
+        $activities = $environment->getActivities(0, null, null, Activity::STATE_STAGED);
+        if (count($activities) < 1) {
+            $output->writeln("Nothing to deploy");
```

Comment:
All messages on stderr

```suggestion
            $this->stdErr->writeln("Nothing to deploy");
```